docs/src/developer/introduction.md

```diff
@@ -0,0 +1,250 @@
+
+```
+git checkout -b mynewbranch
+```
+
+If your contribution is small and does not take a long time to implement,
+everything will likely be fine if you simply commit the changes locally, then
+push them to your GitHub account online:
+
+```
+git commit -a
+git push --all
+```
+
+However, if you are working on a much larger project it is *highly* recommended
+that you frequently pull from the official master branch and rebase your new
+branch on top of any changes that have been made there:
+
+```
+git checkout master
+git pull
+git checkout mynewbranch
+git rebase master
+```
+
+Note that rebasing will try to rewrite each of your commits over the top of the
+branch you are rebasing on (master in this case). This process will have many
+steps if there are many commits and lots of conflicts. Simply follow the
+instructions until the process is finished.
+
+The longer you leave it before rebasing on master the longer the rebase process
+will take. It can eventually become overwhelming as it is not replaying the
+latest state of your repository over master, but each commit that you made in
+order. You may have completely forgotten what those older commits were about,
+so this can become very difficult if not done regularly.
+
+Once you have pushed your changes to your GitHub account, go to the official
+project GitHub page and you should see your branch mentioned near the top of
+the page. Open a pull request.
+
+Someone will review your code and suggest changes they'd like made. Simply
+add more commits to your branch and push again. They will automatically get
+added to your pull request.
+
+Note that we don't accept code without tests and documentation. We use
+Documenter.jl for our documentation, in Markdown format. See our existing
+code for examples of docstrings above functions in the source code and look
+in the `docs/src` directory to see how these docstrings are merged into our
+online documentation.
+
+## Development list
+
+All developers of AbstractAlgebra and Nemo are welcome to write to our
+development list to ask questions and discuss development:
+
+https://groups.google.com/g/nemo-devel
+
+## Reporting bugs
+
+Bugs should be reported by opening an issue (ticket) on the official GitHub
+page for the relevant project. Please state the Julia version being used,
+the machine you are using and the version of AbstractAlgebra/Nemo you are
+using. The version can be found in the Project.toml file at the top level
+of the source tree.
+
+## Development roadmap
+
+AbstractAlgebra has a special roadmap ticket which lists the most important
+tickets that have been opened. If you want to contribute something high value
+this is the place to start:
+
+https://github.com/Nemocas/AbstractAlgebra.jl/issues/492
+
+This ticket is updated every so often.
+
+## Binaries
+
+Binaries of C libraries for Nemo are currently made in a separate repository:
+
+https://github.com/oscar-system/FLINT_jll.jl
```

Comment:
This is not the official version, but just some experiment to try a beta version of 2.7. The official build recipe resides at https://github.com/JuliaPackaging/Yggdrasil. This is the case for flint, arb and antic.

Reply:
Fixed.